scripts/zones/Phomiuna_Aqueducts/npcs/_0rx.lua

```diff
@@ -0,0 +1,39 @@
+-----------------------------------
+--    Area: Phomiuna_Aqueducts
+--  NPC: Oil Lamp - Water (East)
+-- ID: 16888080  !pos 104 -26 77
+-----------------------------------
+require("scripts/globals/missions");
+local ID = require("scripts/zones/Phomiuna_Aqueducts/IDs");
+-----------------------------------
+
+function onTrade(player,npc,trade)
+end;
+
+function onTrigger(player,npc)
+
+    local DoorOffset = npc:getID();
+
+    player:messageSpecial(ID.text.LAMP_OFFSET+2); -- water lamp
+    npc:openDoor(7); -- lamp animation
+
+    local element = VanadielDayElement();
+    -- printf("element: %u",element);
+
+    if (element == 2) then -- waterday
+        if (GetNPCByID(DoorOffset+7):getAnimation() == 8) then -- lamp fire open?
+            GetNPCByID(DoorOffset-2):openDoor(15); -- Open Door _0rk
```

Comment:
Comment says these eastern doors open up _0rk, but I think they _might_ open a different door, and the erroneous comment was a result of a copy-paste :wink:

Reply:
Corrected. It's door _0rl